SaraAlert/SaraAlert
# SARAALERT-1575: Additional Vaccines and Doses

# Description
Jira Ticket: [SARAALERT-1575](https://tracker.codev.mitre.org/browse/SARAALERT-1575)

* Sara Alert has been updated to support the latest CDC guidance surrounding 3-dose vaccines
* Update import to support 3 vaccine doses
* Add support for two additional non-US approved vaccines across the application
* Update import format and guidance to include 3 vaccine doses and new products/name

## Important Changes

`import_controller.rb`
- Create another vaccine based on the additional dose 3 fields

`import_export_constants.rb`
- Add new headers and field names to constants

`vaccines.yml`
- 3 doses valid for all vaccines
- update naming to include tradenames
- new non-US vaccines

`test/`
- Update fixtures with new naming
- Update existing tests that had hard-coded vaccine names with updated names
- Include ability to verify 3rd dose on import 
- Update test import spreadsheets

`public/`
- Update import format guidance
- Update import format examples
- Update import format changelog

## Testing Recommendations

* Create/Import a third dose of a vaccine
* Create/Import with a a vaccine that had a name change/new vaccine
* Filtering/Advanced Filtering on new names and new vaccines

# Checklists

**Submitter:**


@tstrass  :


@ :


@ngfreiter
